### PR TITLE
Added new tool: TheFatRat

### PR DESCRIPTION
**Pull request by mrsmith0x00**
As soon as my PR will be merge, i will correct this to Screetsec repo.

### CODE REVIEW

File: packages/python-names/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# This file is part of BlackArch Linux ( http://blackarch.org ).
+# See COPYING for license details.
+
+pkgname=python-names
+pkgver=0.3.0
+pkgrel=1
+pkgdesc='Random name generator.'
+arch=('any')
+url='https://pypi.org/project/names'
```
Comment (FFY00) [inline packages/python-names/PKGBUILD]: Should use the github url.

https://github.com/treyhunner/names

File: packages/python-names/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+license=('MIT')
+depends=('python')
+makedepends=('python' 'python-setuptools')
+source=("https://files.pythonhosted.org/packages/44/4e/f9cb7ef2df0250f4ba3334fbdabaa94f9c88097089763d8e85ada8092f84/names-$pkgver.tar.gz")
```
Comment (FFY00) [inline packages/python-names/PKGBUILD]: Github url.

File: packages/thefatrat/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+license=('GPL3')
+depends=('searchsploit' 'backdoor-factory' 'metasploit' 'xterm' 'bind-tools' 'gcc'
+         'apache' 'gnome-terminal' 'upx' 'ruby-nokogiri' 'openssl' 'imagemagick'
+         'lib32-ncurses' 'lib32-zlib' 'lib32-libstdc++5' 'lsb-release' 'python' 'python-names'
+         'jdk8-openjdk' 'jre8-openjdk' 'unzip' 'mingw-w64-gcc' 'sudo')
+makedepends=('git')
+source=('git+https://github.com/mrsmith0x00/TheFatRat.git')
+install=thefatrat.install
+sha512sums=('SKIP')
+options=('!strip')
```
Comment (FFY00) [inline packages/thefatrat/PKGBUILD]: Why?

File: packages/thefatrat/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+_dir_name_=TheFatRat
+
+pkgver() {
+  cd "$_dir_name_"
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```
Comment (FFY00) [inline packages/thefatrat/PKGBUILD]: A tag is avaliable. Add it in the `$pkgver` too.

File: packages/thefatrat/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  
+  install -Dm 755 "$pkgdir/usr/bin/"
+  install -Dm 755 "$pkgdir/usr/share/$pkgname"
+
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md CHANGELOG.md troubleshoot.md
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+
+  rm -rf *.md LICENSE
+  
+  cp -a * "$pkgdir/usr/share/$pkgname"
```
Comment (FFY00) [inline packages/thefatrat/PKGBUILD]: Replace `-a` with ``--no-preserve=ownership`.

File: packages/thefatrat/PKGBUILD (hunk 4 above)
Comment (mrsmith0x00) [reply packages/thefatrat/PKGBUILD]: it isn't necessary yet

File: packages/thefatrat/PKGBUILD (hunk 3 above)
Comment (mrsmith0x00) [reply packages/thefatrat/PKGBUILD]: Without this option build produce errors.

File: packages/python-names/PKGBUILD (hunk 1 above)
Comment (mrsmith0x00) [reply packages/python-names/PKGBUILD]: Complete.

File: packages/python-names/PKGBUILD (hunk 2 above)
Comment (mrsmith0x00) [reply packages/python-names/PKGBUILD]: Complete.

**Comment by noptrix** [comment]
thank you guys

**Pull request merged**